docs/source/query.rst

```diff
@@ -155,17 +155,12 @@ Logical Operators
 There are three supported logical operators: ``$and``, ``$or``, and ``$not``.
 The first two are unique in that they involve combinations of other query operators.
 To query with one of these two logical expression, we construct a mapping with the logical operator as the key and a list of expressions as the value.
-As usual, the ``$and`` operator matches documents where all the expressions are true, while the ``$or`` expression matches if any documents satisfy the provided expression.
-For example, we can match all documents where *p is greater than 2* **or** *kT=1.0* we could use the following (split onto multiple lines for clarity):
+As usual, the ``$and`` operator matches documents where all the expressions are true, while the ``$or`` expression matches if documents satisfy any of the provided expressions.
+For example, to find all documents where *p is greater than 2* **or** *kT=1.0*, we could use the following:
```

Comment:
Pretty fancy tool to make these changes :smike

Reply:
You know, you'd be surprised. 😉 This might be coming soon!

https://twitter.com/marlene_zw/status/1509539512422129667
https://github.com/PrithivirajDamodaran/Gramformer